src/components/NewsRoomList.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { useState } from "react";
+import FilterList from "./FilterList";
+import NewsRoomCard from "./NewsRoomCard";
+import { newsItems } from "../files/NewsItems";
+import ListCounter from "./ListCounter";
+import useNewsRoom from "../hooks/useNewsRoom";
+import { Alert, Button } from "@baltimorecounty/dotgov-components";
+
+const NewsRoomList = () => {
+  const { hasError, newsRoomItems = [], isLoading } = useNewsRoom();
+  const [filteredItems, setFilteredItems] = useState([]);
+
+  const handleLoadMoreNewsItems = clickEvent => {
+    setFilteredItems([]);
+    console.log("Load More Results");
+  };
+
+  if (hasError) {
+    return (
+      <Alert className="status" type="error">
+        <p>
+          Unable to retrieve the list of Baltimore County news room. Please try
+          again in a couple of minutes.
+        </p>
+      </Alert>
+    );
+  }
+
+  return (
+    <React.Fragment>
+      {isLoading ? (
+        <p>Loading Baltimore County News...</p>
+      ) : (
+        <div className="row">
+          <div className="col-md-6 col-xs-12 order-xs-last order-md-first ">
+            <ListCounter
+              count={
+                filteredItems.length === 0
+                  ? newsRoomItems.length
+                  : filteredItems.length
+              }
+              total={newsRoomItems.length}
+            />
+          </div>
+          <FilterList
```

Comment:
Tied to the comment above, we probalby don't want this here, but since we know it's coming probably not worth removing. I think this is specified in the acceptance criteria, but will double check.

Reply:
So the acceptance criteria seems to be out of whack entirely with the tasks. The User story acceptance has this entire page working along with load more, displaying full story (redirect to new page and the page to display the info). The only things now supposed to be added is the Share button functionality and the Filtering. I added list counter because its in the XD document and the description of the story but I dont specifically see it in the acceptance criteria. So Ive removed this but I honestly feel that these stories confusing and this process isnt working.

Reply:
@tmccoy529 I reviewed the user story and i agree. I had not seen this since Kevin updated and can clearly see where some of the confusion is coming.

Is the XD document adding to the confusion or no?